.gitignore

```diff
@@ -10,3 +10,11 @@
 *.iml
 /designs/
 interchange/schema/capnp/java.capnp
+*.bit
+vivado*
+*.dcp
+*.edf
+*.edif
+.*
+*.log
+*.bak
```

Comment:
Could we hold off on these changes for now?  Perhaps in a different PR?

Reply:
I believe .gitignore should never been merged. However, each person will have a different .gitignore, we need to find a way to ignore the .gitignore.    I can revert to the version from master.   I hope there is a way to ignore the file during PR merging.